firedrake/petsc.py

```diff
@@ -7,10 +7,17 @@
 from petsc4py import PETSc
 import itertools
 import functools
+import gc
+import warnings
 from contextlib import contextmanager
+from pyop2 import mpi
+from typing import Any
 
 
-__all__ = ("PETSc", "OptionsManager", "get_petsc_variables")
+__all__ = (
+    "PETSc", "OptionsManager", "get_petsc_variables", "garbage_cleanup",
+    "garbage_view",
+)
```

Comment:
```suggestion
__all__ = ("PETSc", "OptionsManager", "get_petsc_variables")
```
lets keep these private.

Reply:
Ooops, meant to apply that suggestion!